app/services/streaming/streaming-view.ts

```diff
@@ -248,10 +248,24 @@ export class StreamInfoView<T extends Object> extends ViewHandler<T> {
 
   /**
    * Chat url of a primary platform
+   * If the primary platform is not enabled, and we're on single stream mode,
+   * returns the URL of the first enabled platform
    */
   get chatUrl(): string {
     if (!this.userView.isLoggedIn || !this.userView.auth) return '';
-    return getPlatformService(this.userView.auth.primaryPlatform).chatUrl;
+    // TODO: transition away from primaryPlatform completely when possible
+    // TODO: handle custom destinations
+    // TODO: handle selected platform not supporting chat
```

Comment:
i dont think these todos are todos anymore:
1. we pivoted to primaryPlatform === primaryChat
2 and 3. we just wouldnt display chat for those